site/routes/definitions.js

```diff
@@ -1,14 +1,27 @@
 // @flow
 
-type RouteDefinition = {|
+type RouteDefinition = {
   title: string | (props: Object) => string,
   key: string,
   route: string,
-  defaults?: any,
   stateToProps?: (state: Object, params?: Object) => any,
   gen?: (state: Object) => Array<Object>,
-  render?: (state: Object) => any,
-|}
```

Comment:
It's nice to keep the exact object type here if possible, it'll prevent you misspelling a key in the config. All the functions can just be optional.